lib/SPIRV/Mangler/ParameterType.h

```diff
@@ -420,6 +420,7 @@ struct UserDefinedType : public ParamType {
 struct TypeVisitor {
   SPIRversion spirVer;
   TypeVisitor(SPIRversion ver) : spirVer(ver){};
+  virtual ~TypeVisitor() {}
```

Comment:
How about compiler generated destructor? 

`virtual ~TypeVisitor() = default;`

Reply:
Right, I forgot about `= default`, that would be better.